modules/ROOT/pages/fd-share-to-studio.adoc

```diff
@@ -30,8 +31,8 @@ A `.jar` file that contains your project source files, including all the connect
 +
 Studio opens the project.
 
-The person who works in Studio can work on the project, and all changes are made automatically to the project in Design Center.
-When the work in Studio is completed, ensure that the project is deleted from Studio. If you need help again later, repeat the procedure to share your project.
+The person who works in Studio can work on the project. When the work in Studio is completed, ensure that the project is deleted from Studio.
```

Comment:
suggest we just kill this line 
> When the work in Studio is completed, ensure that the project is deleted from Studio. If you need help again later, repeat the procedure to share your project.

looks good otherwise

Reply:
Fixed.